google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/HttpStorageRpc.java

```diff
@@ -127,12 +127,14 @@ public HttpStorageRpc(StorageOptions options) {
     private final Storage storage;
     private final LinkedList<BatchRequest> batches;
     private int currentBatchSize;
+    private final LinkedList<RetryBatchCallBack> failedBatches;
```

Comment:
failedBatches is not correct name, I guess you mean failedRpcs

Reply:
Done.